vocabularies/Hierarchy.xml

```diff
@@ -256,184 +242,72 @@ The template actions themselves cannot be invoked.
 
-      <Action Name="Template_CreateAction" IsBound="true" EntitySetPath="HierarchyNodes">
-        <Annotation Term="Common.Experimental" />
-        <Annotation Term="Core.Description" String="Template for actions that create a new node and are named in [`RecursiveHierarchy/CreateAction`](#RecursiveHierarchyType)" />
-        <Parameter Name="HierarchyNodes" Type="Collection(Edm.EntityType)" Nullable="false" />
-        <Parameter Name="HierarchyQualifier" Type="Aggregation.HierarchyQualifier" Nullable="false" />
-        <Parameter Name="Entity" Type="Edm.EntityType" Nullable="false">
-          <Annotation Term="Core.Description" String="The new entity to be created as a node" />
-        </Parameter>
-        <Parameter Name="Parent" Type="Edm.EntityType" Nullable="true">
-          <Annotation Term="Core.Description" String="Key of the new node's parent (null if the new node shall be a root)" />
-        </Parameter>
-        <Parameter Name="NextSibling" Type="Edm.EntityType" Nullable="true">
-          <Annotation Term="Core.Description" String="Key of the new node's next sibling (null if the new node shall be the last sibling)" />
-          <Annotation Term="Core.LongDescription">
-            <String>In the absence of this parameter, the sibling position is determined by the service.</String>
-          </Annotation>
-          <Annotation Term="Core.OptionalParameter">
-            <Record/>
-          </Annotation>
-        </Parameter>
-        <ReturnType Type="Edm.EntityType" Nullable="false">
-          <Annotation Term="Core.Description" String="The new node" />
-          <Annotation Term="Core.LongDescription">
-            <String>The values of the properties declared in the [`RecursiveHierarchy`](#RecursiveHierarchyType) annotation
-            can only be obtained with a subsequent GET request that contains a hierarchical transformation
-            in an `$apply` query option.</String>
-          </Annotation>
-        </ReturnType>
-      </Action>
-      <Action Name="Template_CreateChildAction" IsBound="true" EntitySetPath="HierarchyNodes">
+      <Action Name="Template_LinkAction" IsBound="true">
         <Annotation Term="Common.Experimental" />
-        <Annotation Term="Core.Description" String="Template for instance-bound actions that create a new non-root node and are named in [`RecursiveHierarchy/CreateChildAction`](#RecursiveHierarchyType)" />
+        <Annotation Term="Core.Description" String="Template for actions that link a node to a parent and are named in [`RecursiveHierarchy/LinkAction`](#RecursiveHierarchyType)" />
         <Annotation Term="Core.LongDescription">
-          <String>This action can be used when creating several nodes in one JSON batch request like:
+          <String>In single-parent hierarchies, the node is first unlinked from its current parent, if any.
+
+To create a new node as child of an existing parent, this action can be used in a JSON batch request like:
 ```
```

Comment:
```suggestion
```json
```

Reply:
Not valid JSON, because the code contains ...